package.json

```diff
@@ -4,6 +4,7 @@
   "version": "6.1.0",
   "author": "Bobby Sudekum",
   "main": "./javascript/index.js",
+  "types": "./index.d.ts",
```

Comment:
You don't need to define this when your type file is in the project root and it is named `index.d.ts` 👍 So you can remove this line